### ISSUE TITLE
Allow manually setting version

### ISSUE THREAD

**Issue by jelovirt** [mislav/bump-homebrew-formula-action#8]
GitHub released [manual triggers for workflows](https://github.blog/changelog/2020-07-06-github-actions-manual-triggers-with-workflow_dispatch/) and this allows actions that are not attached to tag events. [Current action code](https://github.com/mislav/bump-homebrew-formula-action/blob/master/src/main.ts#L31..L32) assumes tags will always start with `v` and the version is only read from the tag name.

In order to allow workflows that are manually triggered, input for version should be added. The default value for version should be read from the tag to retain the current functionality.

**Comment by mislav** [comment]
That's an excellent idea! Contributions welcome.

I've just released a version that supports git-based `download-url` for Homebrew formula:
```rb
url "https://github.com/cli/cli.git",
  tag: "v0.11.1",
  revision: "<SHA>"
```

Keep in mind that for updating formulae of this format, a commit SHA that corresponds to the tag name is also necessary as input. Maybe, instead of requiring the SHA as input, we could resolve it from the entered tag name.

**Comment by jelovirt** [comment]
@mislav opened  #9 to implement this. Tested with debug workflow and fork of Homebrew/homebrew-core, the generated PR looks correct.

**Comment by mislav** [comment]
Fixed in 3b71c4122f2c7ea28ab85033c20a893fa08a26e0
